BitwardenShared/Core/Platform/Services/NotificationService.swift

```diff
@@ -308,14 +307,14 @@ class DefaultNotificationService: NotificationService {
         let request = UNNotificationRequest(identifier: data.id, content: content, trigger: nil)
         try await UNUserNotificationCenter.current().add(request)
 
-        // If the request is for the existing account, show the login request view automatically.
-        guard let loginRequest = try await authService.getPendingLoginRequest(withId: data.id).first
```

Comment:
This API request was being made if the login request was for an active or inactive user. And it fails for an inactive user since it has the wrong access token. This instead needs to happen after switching accounts if the request is for an inactive user.